packages/devtools/devtools/src/panels/mesh/SignalMessages.stories.tsx

```diff
@@ -0,0 +1,17 @@
+//
```

Comment:
@mykola-vrmchk i made snippets for this and react components, try writing `story` or `component` in a `tsx` file

Reply:
fyi @wittjosiah